### PR TITLE
Update gsoc2023ideas.md

### PR DESCRIPTION
**Pull request by adeyosemanputra**

### CODE REVIEW

File: pages/initiatives/gsoc/gsoc2023ideas.md [hunk 1]
```diff
@@ -178,3 +178,85 @@ If you have a new Idea regarding Iris(Meta searcher), let us chat.
 ##### Mentors
 
 * [Saeed Dehqan](mailto:saeed.dehghan@owasp.org)
+
+
+### [OWASP SecureTea](https://securetea.org/)
+
+The OWASP SecureTea Project provides a one-stop security solution for various devices (personal computers / servers / IoT devices).
+![Possible for "Medium" GSoC 2022 project](https://img.shields.io/badge/medium%20size%20(~175h)-possible-yellow)
```
Comment (psiinon) [inline pages/initiatives/gsoc/gsoc2023ideas.md:186]: Change 2022 - 2023 everywhere

**Review by psiinon — CHANGES_REQUESTED** [review]
Also need to add link to your project at the top

File: pages/initiatives/gsoc/gsoc2023ideas.md (hunk 1 above)
Comment (adeyosemanputra) [reply pages/initiatives/gsoc/gsoc2023ideas.md:186]: ok

File: pages/initiatives/gsoc/gsoc2023ideas.md [hunk 2]
```diff
@@ -183,13 +183,13 @@ If you have a new Idea regarding Iris(Meta searcher), let us chat.
...
+![Possible for "Medium" GSoC 2023 project](https://img.shields.io/badge/medium%20size%20(~175h)-possible-yellow)
+![Preferred for "Large" GSoC 2023 project](https://img.shields.io/badge/large%20size%20(~350h)-preferred-green)
 ##### Expected results
-![Possible for "Medium" GSoC 2022 project](https://img.shields.io/badge/medium%20size%20(~175h)-possible-yellow)
-![Preferred for "Large" GSoC 2022 project](https://img.shields.io/badge/large%20size%20(~350h)-preferred-green)
+![Possible for "Medium" GSoC 2023 project](https://img.shields.io/badge/medium%20size%20(~175h)-possible-yellow)
+![Preferred for "Large" GSoC 2023 project](https://img.shields.io/badge/large%20size%20(~350h)-preferred-green)
 - Improve Web Application Firewall GUI based on AI 
-- Improve features (IDS,Firewall)
+- Improve all Securetea features (IDS,Firewall)
```
Comment (kingthorin) [inline pages/initiatives/gsoc/gsoc2023ideas.md:192]: Do you want the capital T in there or is it fine?

Comment (adeyosemanputra) [reply pages/initiatives/gsoc/gsoc2023ideas.md:192]: sure will change it

File: pages/initiatives/gsoc/gsoc2023ideas.md [hunk 3]
```diff
@@ -178,3 +178,85 @@ If you have a new Idea regarding Iris(Meta searcher), let us chat.
 ##### Mentors
 
 * [Saeed Dehqan](mailto:saeed.dehghan@owasp.org)
+
```
Comment (kingthorin) [inline pages/initiatives/gsoc/gsoc2023ideas.md:181]:
```suggestion
```

**Review by kingthorin — APPROVED** [review]

**Pull request merged**